src/modules/transaction/api/index.js

```diff
@@ -14,20 +14,12 @@ import { fromRawLsk } from '@token/fungible/utils/lsk';
 import { validateAddress } from 'src/utils/validators';
 import http from 'src/utils/api/http';
 import { getDelegates } from '@dpos/validator/api';
-import { HTTP_PREFIX } from 'src/const/httpCodes';
+import { httpPaths } from '../configuration';
 import {
   desktopTxToElementsTx,
   sign,
 } from '../utils';
 
-const httpPaths = {
```

Comment:
rever changes non related to blocks

Reply:
@soroushm These changes are somewhat tied to blocks because the block details page has transactions so I had to make some changes in the transaction domain